web-app/packages/lib/src/modules/user/views/ProfileViewTemplate.vue

```diff
@@ -104,57 +104,58 @@ SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
     </app-container>
     <app-container>
-      <app-section-banner>
+      <app-section>
         <template #title>Advanced</template>
-        <div
-          :class="[
-            'flex flex-column align-items-start text-sm py-2',
-            'row-gap-2',
-            'md:align-items-center md:flex-row'
-          ]"
-        >
-          <div class="flex-grow-1">
-            <p class="font-semibold my-2">Receive notifications</p>
-            <span class="text-xs opacity-80"
-              >We will send you information about workspace activity</span
-            >
-          </div>
+        <div class="flex flex-column row-gap-3 text-sm px-4 pb-4">
           <div
-            class="flex align-items-center flex-shrink-0"
-            data-cy="profile-notification"
+            :class="[
+              'flex flex-column align-items-start',
+              'row-gap-2',
+              'md:align-items-center md:flex-row'
+            ]"
           >
-            <PInputSwitch
-              :modelValue="loggedUser?.receive_notifications"
-              @change="receiveNotificationsChange"
-            />
-          </div>
-        </div>
-        <div
-          :class="[
-            'flex flex-column align-items-start text-sm py-2',
-            'row-gap-2',
-            'md:align-items-center md:flex-row'
-          ]"
-        >
-          <div class="flex-grow-1">
-            <p class="font-semibold my-2">Close account</p>
-            <span class="text-xs opacity-80"
-              >Your account will be closed. In case you are an owner of a
-              workspace, you might need to transfer the ownership first or close
-              the workspace.</span
+            <div class="flex-grow-1">
+              <p class="font-semibold my-2">Receive notifications</p>
+              <span class="text-xs opacity-80"
+                >We will send you information about workspace activity</span
+              >
+            </div>
+            <div
+              class="flex align-items-center flex-shrink-0"
+              data-cy="profile-notification"
             >
+              <PInputSwitch
+                :modelValue="loggedUser?.receive_notifications"
+                @change="receiveNotificationsChange"
+              />
+            </div>
           </div>
-          <div class="flex-shrink-0">
-            <PButton
-              @click="confirmDeleteUser"
-              severity="danger"
-              data-cy="profile-close-account-btn"
-            >
-              Close account</PButton
-            >
+          <div
+            :class="[
+              'flex flex-column align-items-start',
+              'row-gap-2',
+              'md:align-items-center md:flex-row'
+            ]"
+          >
+            <div class="flex-grow-1">
+              <p class="font-semibold my-2">Close account</p>
+              <span class="text-xs opacity-80"
+                >Your account will be closed. In case you are an owner of a
```

Comment:
This text about workspace ownership should be in EE, here it does not have much sense.

Reply:
discussed with @tomasMizera -> it's ok.